.travis.yml

```diff
@@ -59,6 +59,8 @@ install:
   # googleapis is not Go code, but it's required for .pb.go regeneration because of API dependencies.
   - git clone --depth=1 https://github.com/googleapis/googleapis.git "$GOPATH/src/github.com/googleapis/googleapis"
   - go get ${GOFLAGS} -d -t ./...
+  # TODO(drysdale): remove this once https://github.com/grpc/grpc-go/pull/2733/files#r271705181 is addressed upstream
+  - cd ${GOPATH}/src/google.golang.org/grpc && git revert --no-edit 955eb8a3c829b && cd -
```

Comment:
If it breaks it will be in the wrong directory. Not sure if it exits on error? Or if subshells are allowed in this script. This might be the only option.

Reply:
Shifted to subshell like a few lines above.